packages/cli/src/zosfiles/create/Create.utils.ts

```diff
@@ -33,6 +33,7 @@ export function generateZosmfOptions(commandArguments: Arguments): ICreateDataSe
         mgntclass: commandArguments.managementClass,
         dataclass: commandArguments.dataClass,
         dsntype: commandArguments.dataSetType,
+        showAttributes: commandArguments.showAttributes || commandArguments.attributes,
```

Comment:
Hey @ATorrise, thanks for adding these back.
Please note that since `showAttributes` was removed from the command arguments, the first part of this expression will always be falsy.
That's why I was suggesting to change it to `showAttributes: commandArguments.attributes,` (https://github.com/zowe/zowe-cli/pull/1981#discussion_r1445289440)

Reply:
whoops